.build/traefik/http.yml

```diff
@@ -153,7 +153,7 @@ http:
       rule: 'PathPrefix(`/identity/ory/kratos/admin`)'
```

Comment:
after 0.9.0 kratos admin is under the /admin prefix, so we don't need to strip it anymore, we only strip /identity/ory/kratos now